src/pybind/mgr/volumes/fs/operations/volume.py

```diff
@@ -117,6 +117,102 @@ def delete_volume(mgr, volname, metadata_pool, data_pools):
     result_str = "metadata pool: {0} data pool: {1} removed".format(metadata_pool, str(data_pools))
     return r, result_str, ""
 
+def rename_volume(mgr, volname, newvolname):
```

Comment:
I'd like this method to be type checked in order to make changes to the orchestrator API feasible. 
```suggestion
def rename_volume(mgr, volname: str, newvolname: str) -> Tuple[int, str, str]:
```